### DISCUSSION TITLE
Is there a way to implement Tampering Protection?

### DISCUSSION THREAD

**Discussion by YamacorePriv** [jonaswinkler/paperless-ng#1158] [Support]
Hello, i just wanted to ask if anyone managed to implement a protection against tampering after importing like the DIN norms in the EU sadly require.
Want to get rid of all the paper :P

Greets Thalooka

**Comment by jonaswinkler** [comment]
Paperless keeps MD5 checksums of all documents, but I don't think that will satisfy the norm.

**Reply by Yamacore** [reply]
ah thanks for the answer, maybe this is a featue to consider? :P
